app/assets/javascripts/discourse/app/components/sidebar/common/community-section.js

```diff
@@ -1,89 +1,118 @@
-import Component from "@glimmer/component";
-import { inject as service } from "@ember/service";
+import I18n from "I18n";
+import SectionLink from "discourse/components/sidebar/user/section-link";
+import Composer from "discourse/models/composer";
+import { getOwner } from "discourse-common/lib/get-owner";
 import { tracked } from "@glimmer/tracking";
-
+import { action } from "@ember/object";
+import { next } from "@ember/runloop";
+import PermissionType from "discourse/models/permission-type";
+import EverythingSectionLink from "discourse/lib/sidebar/common/community-section/everything-section-link";
+import MyPostsSectionLink from "discourse/lib/sidebar/user/community-section/my-posts-section-link";
+import AdminSectionLink from "discourse/lib/sidebar/user/community-section/admin-section-link";
+import AboutSectionLink from "discourse/lib/sidebar/common/community-section/about-section-link";
+import FAQSectionLink from "discourse/lib/sidebar/common/community-section/faq-section-link";
+import UsersSectionLink from "discourse/lib/sidebar/common/community-section/users-section-link";
+import GroupsSectionLink from "discourse/lib/sidebar/common/community-section/groups-section-link";
+import BadgesSectionLink from "discourse/lib/sidebar/common/community-section/badges-section-link";
+import ReviewSectionLink from "discourse/lib/sidebar/user/community-section/review-section-link";
 import {
   customSectionLinks,
   secondaryCustomSectionLinks,
 } from "discourse/lib/sidebar/custom-community-section-links";
 
-export default class SidebarCommunitySection extends Component {
-  @service router;
-  @service topicTrackingState;
-  @service currentUser;
-  @service appEvents;
-  @service siteSettings;
-
-  @tracked sectionLinks;
-  @tracked moreSectionLinks;
-  @tracked moreSecondarySectionLinks;
-
-  callbackId;
-  headerActionsIcon;
-  headerActions;
-
-  constructor() {
-    super(...arguments);
-
-    this.moreSectionLinks = this.#initializeSectionLinks(
-      [...this.defaultMoreSectionLinks, ...customSectionLinks],
-      { inMoreDrawer: true }
-    );
-
-    this.moreSecondarySectionLinks = this.#initializeSectionLinks(
-      [
-        ...this.defaultMoreSecondarySectionLinks,
-        ...secondaryCustomSectionLinks,
-      ],
-      { inMoreDrawer: true }
-    );
-
-    this.sectionLinks = this.#initializeSectionLinks(
-      this.defaultMainSectionLinks,
-      { inMoreDrawer: false }
-    );
-
-    this.callbackId = this.topicTrackingState.onStateChange(() => {
-      this.sectionLinks.forEach((sectionLink) => {
-        sectionLink.onTopicTrackingStateChange();
+const LINKS_IN_BOTH_SEGMENTS = ["/review"];
```

Comment:
I think this is confusing because `BOTH_SEGMENTS` is not clear and doesn't represent the distinction between anon and logged in user. Perhaps we can just use two lists here? One for anon and another for logged in users.